repo/packages/C/cassandra/0/resource.json

```diff
@@ -1,9 +1,9 @@
 {
   "assets": {
     "uris": {
-      "cassandra-mesos-0-2-0-1-tar-gz":
+      "cassandra-mesos-binary-tar-gz":
       "https://downloads.mesosphere.com/cassandra-mesos/artifacts/0.2.0-1/cassandra-mesos-0.2.0-1.tar.gz",
-      "jre-7u76-linux-x64-tar-gz":    "https://downloads.mesosphere.com/java/jre-7u76-linux-x64.tar.gz"
+      "jre-tar-gz":    "https://downloads.mesosphere.com/java/jre-7u76-linux-x64.tar.gz"
```

Comment:
If possible please don't modify the keys, we are going to build tools that can generate this file automatically from a directory. I think this change will make it hard to build such tools. This requires that the keys are generated from the filename.